app/account_linking.py

```diff
@@ -182,6 +182,9 @@ def process(self) -> LinkResult:
         )
         Session.commit()
 
+        if not new_user.created_by_partner:
```

Comment:
do you know in what case `new_user.created_by_partner` can be True here?

Reply:
When called from the partner_api this flag will be True